### PR TITLE
SPARQL ant Turtle languages support

### PR DESCRIPTION
**Pull request by cnstntn-kndrtv**
two commits:
- SPARQL ans Turtle languages 
- Changed the key bindings In 'demo/settings_menu.thml' from 'cmd-q' to 'ctrl-q'. Becase cmd+q on Mac quits the browser.

### CODE REVIEW

File: lib/ace/mode/sparql.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+var FoldMode = require("./folding/cstyle").FoldMode;
+
+var Mode = function() {
+    this.HighlightRules = SPARQLHighlightRules;
+    this.foldingRules = new FoldMode();
+};
+oop.inherits(Mode, TextMode);
+
+(function() {
+    // this.lineCommentStart = ""//"";
```
Comment (nightwing) [inline lib/ace/mode/sparql.js]: This probably should be `this.lineCommentStart = "#";` to allow toggleComment command to work.
Are there block comments in sparqle mode?

File: lib/ace/mode/turtle.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+var FoldMode = require("./folding/cstyle").FoldMode;
+
+var Mode = function() {
+    this.HighlightRules = TurtleHighlightRules;
+    this.foldingRules = new FoldMode();
+};
+oop.inherits(Mode, TextMode);
+
+(function() {
+    // this.lineCommentStart = ""//"";
```
Comment (nightwing) [inline lib/ace/mode/turtle.js]: same here.

**Comment by cnstntn-kndrtv** [comment]
i have

**Comment by nightwing** [comment]
thanks, merged.

**Pull request merged**